APMrover2/RC_Channel.cpp

```diff
@@ -223,6 +224,15 @@ void RC_Channel_Rover::do_aux_function(const aux_func_t ch_option, const aux_swi
     case AUX_FUNC::MAINSAIL:
         break;
 
+    // save steering trim
+    case AUX_FUNC::SAVE_TRIM:
+        if (!rover.g2.motors.have_skid_steering() && rover.arming.is_armed() && rover.control_mode == &rover.mode_manual && ch_flag == HIGH) {
+            SRV_Channels::set_trim_to_servo_out_for(SRV_Channel::k_steering);
+            rover.trim_radio();
```

Comment:
As discussed on mumble, my feeling at the moment is that we should remove the "rover.trim_radio()" part but I think you're going to give this a test on the water before we decide.

Also maybe we want to change/remove the requirement that the vehicle is in manual mode for this to work.  If we do restrict the mode, and the user is in the wrong mode then we should probably send a text message saying, "Save Trim must be in X mode" or something similar.

Reply:
I tried a version that removes trim.radio and any mode restriction and ran many trials in the wind today. It works the best in ACRO mode with trims averaging ~+/-5us off from best manual trim...then Steering was next best, with RTL/SRTL/GUIDED/AUTO all being up to 25us off randomly,it can be really off if timed during a maneuver or at the end while loitering ...25us is  noticeably incorrect trim when back in MANUAL....of course LOITER and HOLD are no-nos...MANUAL is a bit of a pain having to reset the trim tab, but its obvious that it has to be done...I have pushed a version with out the trim.radio, but disallowing its use during LOITER and HOLD...when merged I will update wiki with suggestion that to get best results use during ACRO ...